.eslintrc.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   root: true,
   plugins: [
-    '@propelinc',
+    '@typescript-eslint',
```

Comment:
Are these basically copied from freshebt-vue? If so, it may be high time for a `@propel/eslint-config` (though obviously not within the scope of this PR).

Reply:
Along a similar vein, I'd love to see us adopt prettier everywhere too.

Reply:
ooo, yeah!